style.css

```
@@ -472,7 +476,7 @@ main h2, main h3, main h4, main h5, main h6 {
 
 .imports li, code, .decl_header, .attributes, .structure_field,
         .constructor, .instances li, .equation, #search_results div {
-    font-family: 'Source Code Pro', monospace;
+    font-family: 'Source Code Pro', monospace, 'STIX Two Math';
```

```suggestion
    font-family: 'Source Code Pro', 'STIX Two Math', monospace;
```

For reproducibility.  `monospace` is the system-provided font.

---

I was worried that there would be some non-maths characters that would end up showing as non-monospace STIX Math characters, but this doesn't seem to be the case. I'll also make STIX the backup font for normal text, so that, e.g., ℤ prints the same inside and outside source code.